content/docs/tutorials/getting-started-aks-letsencrypt/README.md

```diff
@@ -0,0 +1,812 @@
+
+```bash
+az aks create \
+    --name ${CLUSTER} \
+    --node-count 1 \
+    --node-vm-size "Standard_B2s" \
+    --load-balancer-sku basic
+```
+
+Update your `kubectl` config file with the credentials for your new cluster:
+
+```bash
+az aks get-credentials --admin --name "$CLUSTER"
+```
+
+Now check that you can connect to the cluster:
+
+```bash
+kubectl get nodes -o wide
+```
+
+> ⏲ It will take 4-5 minutes to create the cluster.
+>
+> 💵 To minimize your cloud bill, this command creates a 1-node cluster using a
+> low cost virtual machine and load balancer.
+>
+> ⚠️ This cluster is only suitable for learning purposes it is not suitable for production use.
+>
+> 📖 Read [Run Kubernetes in Azure the Cheap Way](https://trstringer.com/cheap-kubernetes-in-azure/) for more cost saving tips.
+>
+
+## Install cert-manager
+
+Now you can install and configure cert-manager.
+
+Install cert-manager using `helm` as follows:
+
+```bash
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm upgrade cert-manager jetstack/cert-manager \
+    --install \
+    --create-namespace \
+    --wait \
+    --namespace cert-manager \
+    --set installCRDs=true
+```
+
+This will create three Deployments and some Services and Pods in a new namespace called `cert-manager`.
+It also installs various cluster scoped supporting resources such as RBAC roles and Custom Resource Definitions.
+
+You can view some of the resources that have been installed as follows:
+
+```bash
+kubectl -n cert-manager get all
+```
+
+And you can explore the Custom Resource Definitions (cert-manager's API) using `kubectl explain`, as follows:
+
+```bash
+kubectl explain Certificate
+kubectl explain CertificateRequest
+kubectl explain Issuer
+```
+
+> 📖 Read about [other ways to install cert-manager](../../installation/README.md).
+>
+> 📖 Read more about [Certificates and Issuers](../../concepts/README.md).
+
+## Create a test ClusterIssuer and a Certificate
+
+Now everything is ready for you to create your first certificate.
+This will be a self-signed certificate but later we'll replace it with a Let's Encrypt signed certificate.
+
+```yaml
+# clusterissuer-selfsigned.yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: selfsigned
+spec:
+  selfSigned: {}
+```
+🔗 <a href="clusterissuer-selfsigned.yaml">`clusterissuer-selfsigned.yaml`</a>
+
+```bash
+kubectl apply -f clusterissuer-selfsigned.yaml
+```
+
+Then use `envsubst` to substitute your chosen domain name into the following Certificate template:
+
+```yaml
+# certificate.yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: www
+spec:
+  secretName: www-tls
+  privateKey:
+    rotationPolicy: Always
+  commonName: www.$DOMAIN_NAME
+  dnsNames:
+    - www.$DOMAIN_NAME
+  usages:
+    - digital signature
+    - key encipherment
+    - server auth
+  issuerRef:
+    name: selfsigned
+    kind: ClusterIssuer
+```
+🔗 <a href="certificate.yaml">`certificate.yaml`</a>
```

Comment:
suggestion: I don't think it's required for you to do this, but seeing this just triggered a memory for me of one of the benefits of the website migration - embedding YAML files.

See for example the old ACME tutorial (which maybe should be a redirect to your newer/better one, but that's a task for another day) [here](https://github.com/cert-manager/website/blob/90ea038b58d0d9818a775eb66468cae9d733de9d/content/docs/tutorials/acme/nginx-ingress.md?plain=1#L104-L105).

(Which produces [this page](https://cert-manager.io/docs/tutorials/acme/nginx-ingress/#step-4---deploy-an-example-service))

That would ensure that the YAML in this document doesn't get out of sync with the YAML in certificate.yaml.

I like that you've got an explicit link to it as well as embedding it, in any case!

Reply:
Done. In https://github.com/cert-manager/website/pull/1140